Subtitle 1.srt

```diff
@@ -0,0 +1,31 @@
+1
+01:00:18,625 --> 01:00:20,041
+<b>My name is Lucy Nyirenda</b>
+
+2
+01:00:21,208 --> 01:00:23,291
+<b>I stay at Mapalwe Village, Senior group Msiwa</b>

```

Comment:
I'm not sure about capitalization. What does "Senior group" mean?

Reply:
It is a title/ a rank of a chief